### PR TITLE
fix: old hypershift workflow

### PR DESCRIPTION
**Pull request by psturc**

### CODE REVIEW

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

File: ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ chain:
   as: hypershift-aws-destroy-nested-management-cluster
   steps:
   - as: dump-management-cluster
+    optional_on_success: true
```
Comment (psturc) [inline ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml:5]: [more info here](https://docs.ci.openshift.org/docs/architecture/step-registry/#skipping-post-steps-on-success)

File: ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml [hunk 2]
```diff
@@ -52,7 +53,8 @@ chain:
       --name="${CLUSTER_NAME}" \
       --namespace="${HYPERSHIFT_NAMESPACE}" \
       --infra-id=${CLUSTER_NAME} \
-      --base-domain=${DOMAIN}
+      --base-domain=${DOMAIN} \
+      --cluster-grace-period 40m
```
Comment (psturc) [inline ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml:57]: based on [settings in hosted-cluster wf](https://github.com/openshift/release/blob/c0a7ee63273d5f4525534732d3831f736cb51294/ci-operator/step-registry/hypershift/hostedcluster/destroy/hostedcluster/hypershift-hostedcluster-destroy-hostedcluster-commands.sh#L41)

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Comment by flacatus** [comment]
/pj-rehearse ci/rehearse/redhat-appstudio/infra-deployments/main/appstudio-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-application-service-main-application-service-e2e

**Pull request closed without merging**